layers/+distribution/spacemacs-base/packages.el

```diff
@@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
         i.e. `SPC m h h`, to lookup the source of the definition,
         while falling back to `evil-lookup'"
         (interactive)
-        (condition-case nil
+        (condition-case-unless-debug nil
```

Comment:
I realized that afterwards :)
On Thu, Feb 25, 2016 at 5:39 PM Eivind Fonn <notifications@github.com>
wrote:

> In layers/+distribution/spacemacs-base/packages.el
> <https://github.com/syl20bnr/spacemacs/pull/5217#discussion_r54177717>:
>
> > @@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
> >          i.e. `SPC m h h`, to lookup the source of the definition,
> >          while falling back to `evil-lookup'"
> >          (interactive)
> > -        (condition-case nil
> > +        (condition-case-unless-debug nil
>
> I know. I said I agree.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/5217/files#r54177717>.
>